From: Ben Hutchings <ben@decadent.org.uk>
Date: Sun, 02 Apr 2017 01:56:55 +0100
Subject: vfs: Avoid ABI change for "mnt: Add a per mount namespace limit ..."
Forwarded: not-needed
    
Commit d29216842a85c7970c536108e093963f02714498 added two new fields
to struct mnt_namespace.  This structure is not exposed to OOT modules
(as it is defined in fs/mount.h) and is always instantiated in the
core kernel.  Therefore hide the change from genksyms.

---

--- a/fs/mount.h
+++ b/fs/mount.h
@@ -11,8 +11,10 @@ struct mnt_namespace {
 	u64			seq;	/* Sequence number to prevent loops */
 	wait_queue_head_t poll;
 	u64 event;
+#ifndef __GENKSYMS__
 	unsigned int		mounts; /* # of mounts in the namespace */
 	unsigned int		pending_mounts;
+#endif
 };
 
 struct mnt_pcp {
